Lib/test/test_builtin.py

```diff
@@ -1247,6 +1250,106 @@ def test_map_pickle(self):
             m2 = map(map_char, "Is this the real life?")
             self.check_iter_pickle(m1, list(m2), proto)
 
+    def test_map_pickle_strict(self):
```

Comment:
Just wondering but are those tests based on the `zip` tests? if so, that's fine.

Reply:
Yes, I was too lazy to write new tests.